### PR TITLE
Format 'About' section and add 'Experience' section.

### PR DESCRIPTION
**Pull request by indhu-gunda**
Formatted the 'About' section to have a different layout and removed the random fact generator. Created an 'Experience' section with Bootstrap cards for 3 projects.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -47,20 +47,83 @@
...
+        <h1 class="text-center">Experience</h1>
+        <div class="row card-deck">
+          <div class="col-md-4 col-1g-4 col-sm-12 card">
+            <div class="card-header">
+              <h6>Amazon</h6>
+            </div>
+            <div class="card-body">
+              <h4 class="card-title">AFE Software Development Intern</h4>
+              <ul>
+                <li>Built order issue resolution tool for the Supply Chain Optimization Technologies team that summarized order and shipment information from numerous sources and auto resolves issues in certain cases.</li>
```
Comment (J3ffreySmith) [inline portfolio/src/main/webapp/index.html]: Nit: "resolves" should be "resolved"

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -47,20 +47,83 @@
...
+          </div>
+          <div class="col-md-4 col-1g-4 col-sm-12 card">
+            <div class="card-header">
+              <h6>Code.org</h6>
+            </div>
+            <div class="card-body">
+              <h4 class="card-title">Open Source Project</h4>
+              <ul>
+                <li>Creating a feature to facilitate access to pre-reader materials by enabling text to speech to auto play.</li>
+                <li>Building a customizable framework for teachers to employ this feature in their classrooms.</li>
```
Comment (J3ffreySmith) [inline portfolio/src/main/webapp/index.html]: Nit: "Created" and "Built", everything else is past-tense

**Review by J3ffreySmith — APPROVED** [review]

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (indhu-gunda) [reply portfolio/src/main/webapp/index.html]: fixed

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (indhu-gunda) [reply portfolio/src/main/webapp/index.html]: fixed

**Pull request merged**